src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java

```diff
@@ -0,0 +1,17 @@
+package dev.code.digital_lending_microservice.payload.request;
+
+import lombok.Data;
+
+@Data
+public class CreateCustomerDto {
+
+    private String firstName;
```

Comment:
Ensure that firstname and lastname are not null and not empty

Reply:
ok